governance/votes/Executive vote - August 22, 2024.md

```diff
@@ -0,0 +1,142 @@
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then the PSM-USDC-A DC-IAM parameters below will be updated as follows:
+
+- PSM-USDC-A DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be decreased by 7,500 million DAI from 10,000 million DAI to **2,500 million DAI**.
+- PSM-USDC-A DC-IAM [Target Available Debt (`gap`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-) will be decreased by 180 million DAI from 380 million DAI to **200 million DAI**.
+- PSM-USDC-A DC-IAM [Ceiling Increase Cooldown (`ttl`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-) will remain unchanged at **12 hours**.
+
+### Update MCD_LITE_PSM_USDC_A `buf`
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then the MCD_LITE_PSM_USDC_A fixed-sized amount of pre-minted Dai will be increased as follows:
+
+- MCD_LITE_PSM_USDC_A [Fixed-sized Amount of Pre-minted Dai (`buf`)](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512#lite-psm-usdc-a-parameters-overview-8) will be increased by 180 million DAI from 20 million DAI to **200 million DAI**.
+
+### Update LITE-PSM-USDC-A DC-IAM Parameters
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then the LITE-PSM-USDC-A DC-IAM will be updated as follows:
+
+- LITE-PSM-USDC-A DC-IAM [Maximum Debt Ceiling (`line`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be increased by 7,450 million DAI from 50 million DAI to **7,500 million DAI**.
+- LITE-PSM-USDC-A DC-IAM [Target Available Debt (`gap`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-) will be increased by 180 million DAI from 20 million DAI to **200 million DAI**.
+- LITE-PSM-USDC-A DC-IAM [Ceiling Increase Cooldown (`ttl`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-3-ceiling-increase-cooldown-ttl-) will remain unchanged at **12 hours**.
+
+### Update GSM Delay
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
```

Comment:
```suggestion
- **Authorization**: [Governance Facilitator Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/3), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
```

Since this is a Governance Scope item, I believe it should be approved by a Governance Facilitator